bastion/config/config.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/project-nova/backend/pkg/config"
+	"github.com/project-nova/backend/pkg/logger"
+	"github.com/project-nova/backend/pkg/secrets"
+)
+
+type TaskConfig struct {
+	AppID         string `yaml:"app_id"`
+	Region        string `yaml:"region"`
+	DatabaseURI   string `yaml:"database_uri"`
+	ProviderURL   string `yaml:"provider_url"`
+	ApiGatewayUrl string `yaml:"api_gateway_url"`
+}
+
+const (
+	configPath = "/bastion/config"
+)
+
+// GetConfig loads the config
+func GetConfig() (*TaskConfig, error) {
+	var cfg TaskConfig
+
+	path, err := os.Getwd()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get config directories")
```

Comment:
why not logger here?

Reply:
fmt.Errorf is to return an error with type 'error'.